src/TLuaInterpreter.cpp

```diff
@@ -6576,6 +6576,14 @@ int TLuaInterpreter::exists(lua_State* L)
         cnt += host.getAliasUnit()->mLookupTable.count(name);
     } else if (type == "keybind") {
         cnt += host.getKeyUnit()->mLookupTable.count(name);
+    } else if (type == "script") {
```

Comment:
Actually, comparing with literals wrapped with `QLatin1String(`...`)` (or `QStringLiteral(`...`)` if there is any non Latin1 characters and/or parameter replacements, e.g. `%1`, `%2`, etc.) is a better choice if working with `QString`s as then everyone knows that they are not part of the GUI and should be translated by being wrapped in `tr(`...`)`...

Reply:
`mLookupTable.count(QLatin1String(name))`? 
I will fix that in next pr in their respective category for alias, script, keys.

Reply:
:+1: